test/mapmatch.cc

```diff
@@ -1426,16 +1426,17 @@ TEST(Mapmatch, openlr_parameter_true_osrm_api) {
   const auto& matches = response.get_child("matchings");
   EXPECT_EQ(matches.size(), 1);
   const std::vector<std::string>& expected = {
-      "CwOduyULYhtpAAAV//0bGQ==", "CwOdxCULYBtqAAAM//4bGg==", "CwOdySULXxtrAAAf//EbGw==",
-      "CwOd1yULVxtrAQBV/9AbGw==", "CwOduyULYj/gAAACAAA/EA==",
+      "CwOduyULYiKJAAAV//0iGw==",
+      "CwOdxCULYCKJAAAM//4iGw==",
+      "CwOdySULXyKJAAAf//EiGw==",
+      "CwOd1yULVyKLAABV/9AiGw==",
```

Comment:
Why the changed testdata?

Reply:
Oh I see. You fixed the openlr serialization.

Reply:
look at the diff for the file i removed, i explained that there were many bugs